src/jvmMain/kotlin/io/github/fourlastor/wilds_launcher/states/OkDialogState.kt

```diff
@@ -0,0 +1,6 @@
+package io.github.fourlastor.wilds_launcher.states
+
+data class OkDialogState(
+    val lines: Array<String>,
+    val onOk: () -> Unit
```

Comment:
these need to be passed outside of the state due to [recomposition](https://developer.android.com/jetpack/compose/mental-model#recomposition), lambda unfortunately will always fail an equality check